src/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -177,33 +186,29 @@ HRESULT
 HandlerResolver::FindNativeAssemblyFromHostfxr(
     HOSTFXR_OPTIONS* hostfxrOptions,
     PCWSTR libraryName,
-    STRU* struFilename
+    std::wstring& handlerDllPath
 )
 {
-    STRU        struApplicationFullPath;
-    STRU        struNativeSearchPaths;
-    STRU        struNativeDllLocation;
-    INT         intIndex = -1;
-    INT         intPrevIndex = 0;
-    BOOL        fFound = FALSE;
-    DWORD       dwBufferSize = 1024 * 10;
-    DWORD       dwRequiredBufferSize = 0;
-
-    DBG_ASSERT(struFilename != NULL);
+    std::wstring   struNativeSearchPaths;
```

Comment:
Where is the closest try/catch for this function? May want to wrap try/catch around this call?

Reply:
Also, make this return void, HRESULT is never used.

Reply:
in proxymodule, I'll look at it

Reply:
It's used for logging and eventlogs